services/galley/src/Galley/API/Teams.hs

```diff
@@ -978,12 +986,29 @@ deleteTeamMember' lusr zcon tid remove mBody = do
               then 0
               else sizeBeforeDelete - 1
       E.deleteUser remove
-      owners <- E.getBillingTeamMembers tid
-      Journal.teamUpdate tid sizeAfterDelete $ filter (/= remove) owners
+      toNotify <-
+        getFeatureStatus @LimitedEventFanoutConfig DontDoAuth tid
+          >>= ( \case
+                  FeatureStatusEnabled -> E.getBillingTeamMembers tid
+                  FeatureStatusDisabled -> do
+                    let filterFromMembers list =
+                          view userId <$> filter (`hasPermission` SetBilling) (list ^. teamMembers)
+                    filterFromMembers <$> getTeamMembersForFanout tid
+              )
+            . wsStatus
+      Journal.teamUpdate tid sizeAfterDelete $ filter (/= remove) toNotify
```

Comment:
Now that I look at it, maybe this wasn't necessary at all. I'll run the tests in a version where this is left out.

Reply:
Resolved in a commit "Revert the change to the billing team update notification".